Resources/Storage/DIPStorage.py

```diff
@@ -106,8 +106,14 @@ def __putFile( self, src_file, dest_url ):
     localCache = False
     srcDict = res['Value']
     if srcDict['Protocol'] in ['dips', 'dip']:
+      serviceDict = dict( srcDict )
```

Comment:
This is stripping off the file part in the dips url leaving only the service part as needed for the TransferClient. I have added a comment